build-recipe-preinstallimage

```diff
@@ -84,3 +82,23 @@ recipe_cleanup_preinstallimage() {
     :
 }
 
+
+preinstallimage_compress() {
+    local otherdir="$1"
+    local pimage="$otherdir/preinstallimage.tar"
+
+    if test ! -d "$otherdir" -o ! -f "$pimage" -o -L "$pimage" ; then
+        return
+    fi
+
+    case "$(readlink -e -- '$pimage')" in "$(readlink -e -- '$BUILD_ROOT')"*)
```

Comment:
This is a protection in chroot only?

Do we really need it as the build is anyway not safe... ?

Reply:
I think it also protects a symlink being extracted in the VM case. I was concerned that the build result could manipulate something on the build worker host by following symlinks.